client/package.json

```diff
@@ -88,7 +87,6 @@
 		"draft-js": "0.11.3",
 		"email-validator": "2.0.4",
 		"emoji-text": "0.2.6",
-		"emotion-theming": "10.0.19",
```

Comment:
The tests are failing, I _think_, because `emotion-theming` is not available for the tests. Maybe we need to put it into the devDependencies here instead?

Reply:
Thank you, @sirbrillig, that makes me a lot more comfortable!  🙂 I'll keeping look into what exactly is happening with the theming in these tests, then 👍

Reply:
No, I don't think that will help, I'm afraid. Anything in `dependencies` is also available at dev time, and we're not getting any missing module errors. I think the problem is subtler; probably an odd interaction between `emoting-theming` and one of the testing libraries. I'll keep digging!

Reply:
I'm happy to help further, but there's a lot I don't understand about the monorepo and the build process. Please let me know if there's anything I can do!

Reply:
This is an odd one. For some reason, `styled` doesn't add the `theme` prop when running in tests, no matter what I do. This should not be the case, as per the docs:

> The theme prop is automatically injected into components created with `styled`.

However, explicitly wrapping the styled elements with the `withTheme` HOC appears to do the trick, without introducing any issues. It may be redundant, but it should be harmless.

I committed a PR adding the explicit `withTheme` and the tests now pass correctly.

Reply:
Reverted the explicit `withTheme` change, since the problem lied elsewhere.